lib/net.c

```diff
@@ -1171,6 +1190,9 @@ gatekeeper_init_network(struct net_config *net_conf)
 		}
 	}
 
+	/*Randomize RSS key. */
```

Comment:
Please remove the comment, since we know the purpose of the function from its name.